extensions/wikia/Lightbox/LightboxHooks.class.php

```diff
@@ -10,15 +10,7 @@ class LightboxHooks {
 	 * @return bool
 	 */
 	public static function onMakeGlobalVariablesScript( &$vars ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Don't use an OnMakeGlobalVariablesScript hook handler to register JS variables. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AOnMakeGlobalVariablesScript)

[project-id]: wikia:app